integration-tests/elytron-security-oauth2/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-elytron-security-oauth2</artifactId>
+    <name>Quarkus - Integration Tests - Elytron Security OAuth2</name>
+
+    <properties>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-servlet</artifactId>
+                <version>9.4.17.v20190418</version><!-- or 9.4.15.v20190215 -->
```

Comment:
Why do we want Jetty while we have Undertow?

And in any case, it should be in the bom.

Reply:
This is a used by wiremock and there is a dependency management issue in wiremock itself. I can open an issue on wiremock and reference it here to be able to remove this dependency management override later

Reply:
I openned an issue and add the link as a comment in the pom

Reply:
The issue is not on wiremock, it's because we define some jetty dependencies in the bom and not all. I added the missing one.